.github/workflows/precommithooks.yaml

```diff
@@ -0,0 +1,22 @@
+name: pre-commit-hooks
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
```

Comment:
Why is this needed? Can it not use the GITHUB_TOKEN that is provided by default?